x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_explorer/analytics_collection_explorer_table.tsx

```diff
@@ -0,0 +1,327 @@
+        name: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.exploreTable.count',
+          { defaultMessage: 'Count' }
+        ),
+        sortable: true,
+        truncateText: true,
+      },
+    ],
+    sorting: {
+      sort: {
+        direction: 'desc',
+        field: ExploreTableColumns.count,
+      },
+    },
+  },
+  [ExploreTables.TopReferrers]: {
+    columns: [
+      {
+        field: ExploreTableColumns.page,
+        name: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.exploreTable.page',
+          { defaultMessage: 'Page' }
+        ),
+        render: (euiTheme: UseEuiTheme['euiTheme']) => (value: string) =>
+          (
+            <EuiText size="s" color={euiTheme.colors.primaryText}>
+              <p>{value}</p>
+            </EuiText>
+          ),
+        sortable: true,
+        truncateText: true,
+      },
+      {
+        align: 'right',
+        field: ExploreTableColumns.sessions,
+        name: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.collectionsView.exploreTable.session',
+          { defaultMessage: 'Session' }
+        ),
+        sortable: true,
+        truncateText: true,
+      },
+    ],
+    sorting: {
+      sort: {
+        direction: 'desc',
+        field: ExploreTableColumns.sessions,
+      },
+    },
+  },
+};
+
+export const AnalyticsCollectionExplorerTable = () => {
+  const { euiTheme } = useEuiTheme();
+  const { onTableChange, setSelectedTable, setSearch } = useActions(
+    AnalyticsCollectionExploreTableLogic
+  );
+  const { items, isLoading, pageIndex, pageSize, search, selectedTable, sorting, totalItemsCount } =
+    useValues(AnalyticsCollectionExploreTableLogic);
+  let table = selectedTable !== null && (tableSettings[selectedTable] as TableSetting);
+  if (table) {
+    table = {
+      ...table,
+      columns: table.columns.map((column) => ({
+        ...column,
+        render: column.render?.(euiTheme),
+      })) as TableSetting['columns'],
+      sorting: { ...table.sorting, sort: sorting || undefined },
+    };
+  }
+  const handleTableChange = ({ sort, page }: Criteria<ExploreTableItem>) => {
+    onTableChange({ page, sort });
+  };
+
+  useEffect(() => {
+    if (!selectedTable) {
+      const firstTabId = tabs[0].id;
+
+      setSelectedTable(firstTabId, (tableSettings[firstTabId] as TableSetting)?.sorting?.sort);
+    }
+  }, []);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="l">
+      <EuiTabs>
+        {tabs?.map(({ id, name }) => (
+          <EuiTab
+            key={id}
+            onClick={() => setSelectedTable(id, (tableSettings[id] as TableSetting)?.sorting?.sort)}
+            isSelected={id === selectedTable}
+          >
+            {name}
+          </EuiTab>
+        ))}
+      </EuiTabs>
+
+      {table && (
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiSearchBar
```

Comment:
Better to use [Eui search field](https://elastic.github.io/eui/#/forms/form-controls#search-field) instead.